pkgs/by-name/cp/cppman/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  groff,
+  nix-update-script,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "cppman";
+  version = "0.5.9";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aitjcize";
+    repo = "cppman";
+    tag = version;
+    hash = "sha256-iPJR4XAjNrBhFHZVOATPi3WwTC1/Y6HK3qmKLqbaK98=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    wheel
+    distutils
+  ];
+
+  dependencies = [
+    python3Packages.beautifulsoup4
+    python3Packages.html5lib
+    python3Packages.lxml
+    python3Packages.six
+    python3Packages.soupsieve
+    python3Packages.typing-extensions
+    python3Packages.webencodings
+    groff
+  ];
+
+  # cppman pins all dependency versions via requirements.txt as install_requires, causing this check to fail.
+  dontCheckRuntimeDeps = true;
+
+  pythonImportsCheck = [
+    "cppman"
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "C++ 98/11/14 manual pages for Linux/MacOS";
```

The description may be somewhat misleading as the package does not contain the manual pages themselves, but a fetcher / generator / viewer, which, as far as I can tell, is not even limited to Linux and Mac OS.

```suggestion
    description = "Terminal viewer for C++ 98/11/14 manual pages";
```